app/src/main/java/com/fivegmag/a5gmsmediastreamhandler/consumptionReporting/ConsumptionReportingController.kt

```diff
@@ -65,6 +62,79 @@ class ConsumptionReportingController(
         }
     }
 
+    /**
+     * MSISDN = CC + NDC + SN
+     *
+     */
+    @SuppressLint("Range")
+    private fun getMSISDN(): String {
```

Comment:
Add `@RequiresApi(Build.VERSION_CODES.TIRAMISU)` , `getPhoneNumber` is only available with Android API 33

Reply:
Add this to all functions which are related to this.